packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx

```diff
@@ -1,6 +1,339 @@
 ---
 title: Livestream Tutorial
-description: How to build a livestream experience using Stream's video SDKs
+description: How to build a live stream experience using Stream's React Video SDKs
 ---
 
-TODO
+## Quickstart
+In this quickstart, we'll build a live streaming experience that's similar to Twitch.
+
+- The live stream will run on Stream's Edge network of servers around the world
+- You can use ultra-low latency (WebRTC) based live streaming or HLS (slight delay, but better buffering)
+- We'll show how to publish to the live stream from your browser
+- Stream enables you to scale to millions of viewers
+- You can have multiple active participants in the live stream
+
+The app that we are going to build would work in two modes: Host (presenter) and Viewer (audience).
+
+The host will be able to publish the live stream and the viewers will be able to watch the live stream.
+If you want to see a fully-fledged example, check out this [sample app.](https://github.com/GetStream/stream-video-js/tree/main/sample-apps/react/livestream-app)
```

Comment:
```suggestion
If you want to see a full-blown example, check out this [sample app.](https://github.com/GetStream/stream-video-js/tree/main/sample-apps/react/livestream-app)
```

Reply:
I am thinking, that with so many apps we provide in our tutorials, we could introduce an element / banner that would indicate clearly that this is the accompanying sample app / source code. WDYT? Speaking about being more visual and make it easier for readers to find it / determine, that accompanying src code exists.

Reply:
@MartinCupela, I think this is a good idea. I'm sure other SDKs can benefit from such elements as they also provide sample apps like ours.
I'm not sure how much effort would be required to develop such a component, therefore, I'd leave it out for better days :)